native_client/javascript/Makefile

```diff
@@ -2,7 +2,7 @@ NODE_BUILD_TOOL    ?= node-pre-gyp
 NODE_ABI_TARGET    ?= 
 NODE_BUILD_VERBOSE ?= --verbose
 NPM_TOOL           ?= npm
-PROJECT_NAME       ?= deepspeech
+PROJECT_NAME       ?= mozilla_voice_stt
```

Comment:
this makes me thinking, we used to `-gpu` or `-tflite`, are we going to `_gpu` or `_tflite` ?

Also, side note, but sicne we are breaking compat, can we name `cuda` instead of `gpu` ? We can still hope to have OpenCL or ROCm support, this last one seems to have got some traction on TensorFlow side

Reply:
Currently we're sticking with `-gpu` and `-tflite`. I don't have strong opinions on going either way.